<?xml version="1.0"?>

<launch>
  <node name="estimator" type="ekf_node" pkg="roscopter" output="screen">
    <remap from="imu" to="imu"/>
    <remap from="baro" to="dummy_topic"/>
    <remap from="ublox_gnss" to="PosVelEcef"/>
    <remap from="ublox_relpos" to="compass/RelPos"/>
    <remap from="common_ref_lla" to="/ref_lla"/>
  </node>
</launch>
